### PR TITLE
RADOS: TFA fix for slow_osd_heartbeat & CBT/COT Library improvements

### PR DESCRIPTION
**Pull request by harshkumarRH**
# Description

[RHCEPHQE-17076](https://issues.redhat.com/browse/RHCEPHQE-17076) - TFA fix for issue introduced by PR #4241

[RHCEPHQE-16924](https://issues.redhat.com/browse/RHCEPHQE-16924) - Improve ceph-bluestore-tool and ceph-objectstore-tool libraries to control OSD start and stop

Update the crush rule for stretch cluster to remove unsupported entries

Signed-off-by: Harsh Kumar <hakumar@redhat.com>

<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: ceph/rados/bluestoretool_workflows.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ class BluestoreToolWorkflows:
     Contains various functions to verify ceph-bluestore-tool commands
     """
 
-    def __init__(self, node: CephAdmin):
+    def __init__(self, node: CephAdmin, nostop=None, nostart=None):
```
Comment (pdhiran) [inline ceph/rados/bluestoretool_workflows.py:36]: please add a safety check such that both nostop & nostart vars cannot be sent as true for a single object.

File: ceph/rados/objectstoretool_workflows.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ class objectstoreToolWorkflows:
     Contains various functions to verify ceph-objectstore-tool commands
     """
 
-    def __init__(self, node: CephAdmin):
+    def __init__(self, node: CephAdmin, nostop=None, nostart=None):
```
Comment (pdhiran) [inline ceph/rados/objectstoretool_workflows.py:36]: please add a safety check such that both nostop & nostart vars cannot be sent as true for a single object.

File: ceph/rados/bluestoretool_workflows.py (hunk 1 above)
Comment (harshkumarRH) [reply ceph/rados/bluestoretool_workflows.py:36]: Why can't both be set to true? They are mutually exclusive

**Review by pdhiran — APPROVED** [review]
looks good

**Pull request merged**